tests/foreman/ui/test_csv_export.py

```diff
@@ -0,0 +1,781 @@
+"""Test for UI CSV EXPORT
+
+:Requirement: UICSVExport
+
+:CaseLevel: Acceptance
+
+:CaseComponent: UI
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:CaseAutomation: notautomated
+
+:Upstream: No
+"""
+
+from robottelo.decorators import stubbed, tier1
+from robottelo.test import UITestCase
+
+
+class CSVExportHostsTestCase(UITestCase):
+
+    @stubbed()
+    @tier1
+    def test_positive_single_page(self):
+        """
```

Comment:
Usually we include a description of test as the beggining of the doctest as in example: https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/ui/test_activationkey.py#L138

```py
def test_positive_export_with_single_page(self):
    """Assure the exported CSV has all rows when exporting a single page.
    :id: be5e2b2d-3a93-4803-8c6e-d81a97c5a1a5
    :steps:
     .....
    """
```

And looks like all other cases are missing that descriptions